tests/Feature/Service/StudentDetailsServiceTest.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\Feature\Service;
+
+use Tests\TestCase;
+use App\Models\Resume;
+use App\Service\StudentDetailsService;
+use App\Exceptions\StudentNotFoundException;
+
+class StudentDetailsServiceTest extends TestCase
+{
```

Comment:
In this test, I recommend using ``DatabaseTransactions``.

Reply:
Done